test/client/fixtures/automation/click-test.js

```diff
@@ -775,6 +776,99 @@ $(document).ready(function () {
         }, 1500);
     });
 
+    asyncTest('Should raise click event except Firefox if target element appends child after mousedown (GH-423)', function () {
```

Comment:
Please don't use qunit to test automations since we have functional tests.
Try to keep the rule: qunit - for unit testing, testcafe test (our functional test harness) - for functional testing. 

This fix relates to automation sources, they are common for es-next and legacy api. So you should just add a functional test for es-next api